docs.md

```diff
@@ -19,7 +19,7 @@ All pipelines must adhere to the following:
     * Please note the coding guidelines below
 * An [MIT licence](https://choosealicense.com/licenses/mit/)
 * Software bundled using [Docker](https://www.docker.com/)
-    * This must be a build script in the repository (a `Dockerfile`)
+    * This must be at least one building insctructions file in the repository (a `Dockerfile`)
```

Comment:
`insctructions` > `instructions`

How about about _"One or more `Dockerfile`s in the repository"_  to keep it simpler?